src/libpgmoneta/workflow.c

```diff
@@ -131,6 +131,9 @@ wf_backup(void)
    current->next = pgmoneta_workflow_create_permissions(PERMISSION_TYPE_BACKUP);
    current = current->next;
 
+   current->next = pgmoneta_workflow_create_sha256();
```

Comment:
Lets stick it under `if (config->storage_engine == STORAGE_ENGINE_SSH)` - local doesn't need it atm